src/components/dashboard/ColdMap.vue

```diff
@@ -80,12 +71,23 @@ const clusters = computed(() => {
 const tooltipSignal = computed(() => {
   return `{
     title: datum.properties.name,
-    'Gap among ${props.fillStat.name.toLowerCase()}': datum.properties.${
-    props.fillStat.tooltip
-  }
+    'Gap among ${focusFields.value.name.toLowerCase()}': datum.properties.${
+    focusFields.value.population
+  } > 0 ? datum.properties.${
+    focusFields.value.tooltip
+  } : 'Not enough information'
```

Comment:
no more `NA_TEXT`?

Reply:
Honestly, I thought about that when I was doing this PR. I took it out of #105 because only one place used it. And then this PR's code doesn't include #105 which means again, it would only be used in one place. 😆  What have I done Mary. It's okay haha...